game/plugins/src/main/kotlin/gg/rsmod/plugins/content/areas/burthorpe/anton.plugin.kts

```diff
@@ -0,0 +1,94 @@
+package gg.rsmod.plugins.content.areas.burthorpe
+
+import gg.rsmod.plugins.content.mechanics.shops.CoinCurrency
+
+/**
```

Comment:
Replace Alycia's name with yours

Reply:
> Replace Alycia's name with yours

is way auto add my name then jw ? on inteji